backtrace-library/src/main/java/backtraceio/library/services/BacktraceMetrics.java

```diff
@@ -155,11 +158,17 @@ public void enable(BacktraceMetricsSettings settings) {
     }
 
     public void enable(BacktraceMetricsSettings settings, String uniqueEventName) {
+        if (!settings.isBacktraceServer()) {
```

Comment:
lets also check if metrics are already enabled. Lets avoid a situation when we enable metrics twice.